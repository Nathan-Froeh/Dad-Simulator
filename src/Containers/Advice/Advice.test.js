import React from 'react';
import {shallow} from 'enzyme';
import {
  Advice,
  mapDispatchToProps,
  mapStateToProps
} from './Advice';


describe('Advice', () => {
  let wrapper, instance
  const props = {
    getRandomAdvice: jest.fn(),
    getSearchAdvice: jest.fn(),
    randomAdvice: 'randomAdvice',
    searchAdvice: 'searchAdvice'
  }
  beforeEach(() => {
    wrapper = shallow(<Advice {...props}/>)
    instance = wrapper.instance()
  })

  it('should match snap shot', () => {
    expect(wrapper).toMatchSnapshot()
  })

})

