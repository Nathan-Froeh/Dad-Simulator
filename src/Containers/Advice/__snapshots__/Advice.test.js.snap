// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Advice should match snap shot 1`] = `ShallowWrapper {}`;
